src/04-modality-specific-files/08-genetic-descriptor.md

```diff
@@ -0,0 +1,60 @@
+# Genetic Descriptor
+
+The extension was primarily developped by Cyril Pernet and Clara Moreau with contributions from Tom Nichols and Jessica Turner.
+
+The goal of the genetic descriptor is to link imaging and genetic data by providing minimal information about i) where to find genetic information associated with the imaging data and ii) what type of genetic information is available. This is necessary as genetic data are typically stored in dedicated repositories, separately from the imaging data.
```

Comment:
I suggest to revise this paragraph to use a markdown list instead of the in-text i) ii) notation

Reply:
The goal of the genetic descriptor is to link imaging and genetic data. This is necessary as genetic data are typically stored in dedicated repositories, separately from the imaging data. The descriptor provides basics information about:  
* where to find genetic information associated with the imaging data  
* what type of genetic information is available